### ISSUE TITLE
create_lines_with_colors only creates one line with colors.

### ISSUE THREAD

**Issue by pvcraven** [pythonarcade/arcade#804]
This only draws one line:

``` python
import arcade

SCREEN_WIDTH = 800
SCREEN_HEIGHT = 600
SCREEN_TITLE = "Starting Template"


class MyGame(arcade.Window):

    def __init__(self, width, height, title):
        super().__init__(width, height, title)

        arcade.set_background_color(arcade.color.WHITE)

        point_list = [[0, 0],
                      [100, 100],
                      [200, 0],
                      [300, 100]]
        color_list = [arcade.color.RED,
                      arcade.color.GREEN,
                      arcade.color.BLUE,
                      arcade.color.YELLOW]
        self.lines = arcade.create_lines_with_colors(point_list,
                                                     color_list,
                                                     line_width=5)

    def on_draw(self):
        arcade.start_render()
        self.lines.draw()


def main():
    game = MyGame(SCREEN_WIDTH, SCREEN_HEIGHT, SCREEN_TITLE)
    arcade.run()


if __name__ == "__main__":
    main()
```

**Comment by pvcraven** [comment]
<img width="601" alt="screen" src="https://user-images.githubusercontent.com/1463116/102025899-742a0e00-3d60-11eb-8f9b-b06fba928e21.png">

**Comment by einarf** [comment]
Fixed in 745b1e1ec7a041543438cc4ce5f1713c31bf0ec8. It was just an indentation bug 😄

**Comment by pvcraven** [comment]
Closing. Fix will go out in 2.5.1 planned for today.
